src/Umbraco.Web/Media/EmbedProviders/AbstractOEmbedProvider.cs

```diff
@@ -20,7 +25,7 @@ public virtual bool SupportsDimensions
 
         public abstract string GetMarkup(string url, int maxWidth, int maxHeight);
 
-        public virtual string BuildFullUrl(string url, int maxWidth, int maxHeight)
+        public virtual string BuildRequestUrl(string url, int maxWidth, int maxHeight)

```

Comment:
Unfortunately things like this are breaking API changes which we cannot do, are you able to adjust this code so there are no breaking changes?